frontend/src/pages/pipelines/global/pipelines/PipelinesPageHeaderActions.tsx

```diff
@@ -1,21 +1,32 @@
 import * as React from 'react';
-import { Dropdown, DropdownItem, DropdownSeparator, DropdownToggle } from '@patternfly/react-core';
+import { Dropdown, DropdownItem, DropdownSeparator, MenuToggle } from '@patternfly/react-core';
 import { DeleteServerModal, ViewServerModal } from '~/concepts/pipelines/context';
 
-const PipelinesPageHeaderActions: React.FC = () => {
+type PipelinesPageHeaderActionsProps = {
+  toggleElement: React.ReactNode;
+  variant: 'plain' | 'secondary';
+};
+
+const PipelinesPageHeaderActions: React.FC<PipelinesPageHeaderActionsProps> = ({
```

Comment:
Should we still use the name `PipelinesPageHeaderActions` now that it is being used in the `PipelinesSection` page too?

Reply:
A better name IMO can be `PipelineServerActions` (same as the name in the header dropdown on the "Global Pipelines Page").